### DISCUSSION TITLE
'OsmAndCore/QtExtensions.h' file not found

### DISCUSSION THREAD

**Discussion by saigovind4** [osmandapp/OsmAnd#12457] [Q&A]
I tried building the project from https://github.com/osmandapp/OsmAnd-ios.git
I get this issue `'OsmAndCore/QtExtensions.h' file not found`

<img width="1092" alt="Screenshot 2021-08-10 at 7 48 10 PM" src="https://user-images.githubusercontent.com/25867994/128980804-2621ceff-5d5a-4e73-8a7a-5a6804cfc45e.png">

**Comment by vshcherb** [comment]
Possibly this could help. Pure clean installation and compilation works https://github.com/osmandapp/OsmAnd-iOS/actions/runs/1120120661. 
Here you could see definition https://github.com/osmandapp/OsmAnd-iOS/blob/master/.github/workflows/xcode-build.yml

**Reply by GovindB-WAL** [reply]
Thanks, works like charm

**Comment by PaulStets** [comment]
Also, please make sure you run ```./prepare.sh``` before building the project
More info: https://docs.osmand.net/en/main@latest/development/build-osmand/how-to-compile-the-ios-version

**Reply by GovindB-WAL** [reply]
Thanks, works like charm
